.github/CODEOWNERS

```diff
@@ -502,6 +502,11 @@ tests/sentry/api/endpoints/test_organization_dashboard_widget_details.py     @ge
 /static/app/views/insights/pages/platform/                                       @getsentry/telemetry-experience
 ## End of Telemetry Experience
 
+## ML & AI
+*autofix*.py                                                @getsentry/machine-learning-ai
+/static/app/components/events/autofix/                      @getsentry/machine-learning-ai
+/src/sentry/seer/                                           @getsentry/machine-learning-ai
```

Comment:
This was preventing `/src/sentry/seer/similarity/  @getsentry/issue-detection-backend` from matching.